modules/eventstream/src/rpc.c

```diff
@@ -14,7 +14,7 @@
 GglError eventstream_get_common_headers(
     EventStreamMessage *msg, EventStreamCommonHeaders *out
 ) {
-    int32_t message_type = -1;
+    int32_t message_type = 0;
```

Comment:
Doesn't this mean we now default to APPLICATION_MESSAGE when we dont get a header?
Probably fine to do that.